src/schemas/validation/schema.yaml

```
@@ -121,6 +121,8 @@ $defs:
         type: string
       description:
         type: string
+      name:
+        type: [string, null]
```

The `type` keyword requires a single string or an array of strings, and `null` is not a string. You probably meant `type: [string, "null"]` as YAML shortcut for `"type": ["string", "null"]`.

Anyway, I think an explicit value of `null` for the `name` field does not make sense. If I don't want to assign a `name`, I can simply omit the field, same as for the `description`:
```suggestion
        type: string
```